meta-resin-common/classes/kernel-resin.bbclass

```diff
@@ -133,6 +133,7 @@ RESIN_CONFIGS[docker] ?= " \
     CONFIG_EXT4_FS_SECURITY=y \
     CONFIG_AUFS_FS=y \
     CONFIG_AUFS_XATTR=y \
+    CONFIG_KEYS=y \
```

Comment:
That is strange, I have it enabled in a rpi3 build. Is this image using docker 17 or 1.10?